apps/judicial-system/backend/src/app/modules/file/guards/limitedAccessViewCaseFile.guard.ts

```diff
@@ -65,14 +66,13 @@ export class LimitedAccessViewCaseFileGuard implements CanActivate {
       }
     }
 
-    if (isPrisonSystemUser(user)) {
-      if (
-        isCompletedCase(theCase.state) &&
-        caseFile.category &&
-        caseFile.category === CaseFileCategory.APPEAL_RULING
-      ) {
-        return true
-      }
+    if (
+      caseFile.category &&
+      isCompletedCase(theCase.state) &&
+      isPrisonSystemUser(user) &&
```

Comment:
This would give prison staff permission to open files. I suggest creating a new methos `isPrisonAdminUser` in the type lib and use it here instead of this method which includes all prison system users. Also, I suggest renaming the new category list used below to reflect that it is a list of files intended for prison admin users only.

Reply:
Thaaaaanks! Good catch!